### ISSUE TITLE
missing comma in arguments

### ISSUE THREAD

**Issue by johanvos** [jtulach/bck2brwsr#24]
The following code will generate javascript code that misses a comma between the `this` argument and the first parameter:
``` 
       setOnKeyPressed(new EventHandler<KeyEvent>() {
            @Override public void handle(KeyEvent ke) {
                switch (ke.getCode()) {
                    case LEFT:
                        if (getEffectiveNodeOrientation() == NodeOrientation.RIGHT_TO_LEFT) {
                            processRightKey(ke);
                        } else {
                            processLeftKey(ke);
                        }
                        break;
```
The `processRightKey(ke)` is converted into the following:

```
(refs_com_sun_javafx_scene_control_ContextMenuContent || (refs_com_sun_javafx_scene_control_ContextMenuContent = com_sun_javafx_scene_control_ContextMenuContent(false))).processRightKey__VLjavafx_scene_input_KeyEvent_2.call(lcA0['fld_com_sun_javafx_scene_control_ContextMenuContent$1_this$0']lcA1);
```

Manually changing this in 
```
(refs_com_sun_javafx_scene_control_ContextMenuContent || (refs_com_sun_javafx_scene_control_ContextMenuContent = com_sun_javafx_scene_control_ContextMenuContent(false))).processRightKey__VLjavafx_scene_input_KeyEvent_2.call(lcA0['fld_com_sun_javafx_scene_control_ContextMenuContent$1_this$0'],lcA1);
```

works

**Comment by jtulach** [comment]
f786057 tries to (unsuccessfully) simulate the problem. Can you modify the test to match the problematic code?

**Comment by johanvos** [comment]
Done, see https://github.com/jtulach/bck2brwsr/pull/25

**Comment by jtulach** [comment]
Fixed by #25 and 5098a621c1c115da5a4466bc0ea800f5540a5e88
